WorkingAtBabylon.md

```diff
@@ -0,0 +1,89 @@
+<p align="center">
+<img src="logo.png">
+</p>
+
+
+Working in the iOS team
+==================================
+
+TODO
+
+This is more or less modelled after https://github.com/codeOfRobin/Questions-to-ask-your-interviewer-iOS
+
+### Where do we store our code?
+We use Github. We have a central repo that hosts the main and white label apps, and a few Open Source projects that we created and rely on.
+
+## Continuous Integration
+### How many apps/targets/schemes?
+### How big is your Fastfile?
+It's around 250 lines of code at the moment, with about 15-20 lanes for beta testing, daily and custom builds, etc.
+
+### Does it have several environment references?
+### Can you dry run a lane on your machine right now?
+Yes.
+
+### Codesigning?
+  
+* What architectural patterns are you using to make development easier or more consistent? What problems with your current approach have you come across after settling on one (if you have one)?
+
+## Team/People
+### comms - slack?
+Slack, as a non-trivial part of the team is remote. Zoom for meetings of more than 2-3 people.
+
+### Do people ever take sick days to get work done?
+No.
+
+### When was the last time someone worked a late night/weekend in order to meet a deadline?
+TODO
+
+### How many meetings do you have in a day?
+It depend on the squad you’re assigned. For Enrolment and Integrity I counted 18 in a typical sprint, so that averages to ~2 a day. Note that quite a few of them are optional.
+
+### Does your calendar look like a losing game of tetris? Do you have large continuous uninterrupted hours?
+Most of the meetings are on the first Monday/Tuesday of the sprint.
```

Comment:
This is totally squad dependent. In Triage UI for example, this is not the case. It's meetings all over the place.

Reply:
Same i'm afraid

Reply:
```suggestion
It depends a lot on the squad. For Enrolment and Integrity, for example, most meetings are on the first Monday and Tuesday of the sprint. For Triage and Health Management this is not the case and are more spread out.
```